pkg/daemon/config.go

```diff
@@ -30,10 +31,11 @@ type ptp4lConfSection struct {
 }
 
 type ptp4lConf struct {
-	sections     []ptp4lConfSection
-	mapping      []string
-	profile_name string
-	clock_type   event.ClockType
+	sections       []ptp4lConfSection
+	mapping        []string
+	profile_name   string
+	clock_type     event.ClockType
+	gnssSerialPort string // gnss serial port
```

Comment:
Can you keep underscore instead of camelCase to be consistent with other items in ptp4lConf?

Reply:
or you can change the other 2 to use camelCase too

Reply:
done